src/jubilation-axis/x-axis.jsx

```diff
@@ -11,26 +12,43 @@ type Props = {
   title?: string,
   position?: number,
   numTicks?: number,
-  tickLines?: boolean,
+  extendTicks?: boolean,
   axisLine?: boolean
 };
 type Context = { JubilationContext: JubilationContext };
 
 export default function XAxis(
-  { min, max, title, position = 0, numTicks = 0, tickLines = false, axisLine = false }: Props,
+  { min, max, title, position = 0, numTicks = 0, extendTicks = false, axisLine = false }: Props,
   { JubilationContext }: Context
   ): React.Element<*> {
   const context = getContext(JubilationContext);
   const computedMin = min || min === 0 ? min : context.xScale.domain()[0];
   const computedMax = max || max === 0 ? max : context.xScale.domain()[1];
-  const ticks = getTicks(computedMin, computedMax, numTicks, 'x', position, context);
+  const ticks = getTicks({
+    min: computedMin,
+    max: computedMax,
+    numTicks,
+    axisType: 'x',
+    position,
+    context,
+  });
   const offset = context.theme.labelStyle.fontSize;
 
   return (
-    <Animation data={{ min: computedMin, max: computedMax, position, offset, ticks }}>
-      {data =>
+    <Animation data={{ min: computedMin, max: computedMax, position, offset, ticks, extendTicks }}>
+      {(data) =>
         <g>
-          {/* tick values */}
+          {
+            data.extendTicks && data.ticks.map((tick) =>
+              <Line
+                x1={tick.x}
+                x2={tick.x}
+                y1={data.min}
```

Comment:
I think the idea is to have the ticks show up as either something pretty minimal and close to the label ( the red lines in the below diagram) or as spanning the chart (red + green lines below). I can think of three cases we might want

1. show no ticks at all
2. show just the minimal (red) ticks
3. show the extended Ticks (green)

Maybe we should stand up another prop called `showTicks` which would make those situations

1. showTicks = false, extendTicks = whoCares
2. showTicks = true, extendTicks = false
3. showTicks = true, extendTicks = true

![diagram](https://cloud.githubusercontent.com/assets/408448/21077822/3bc0084c-bf26-11e6-8222-c0e2e19bdd62.png)